sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -839,12 +839,14 @@ public partial class ShareFileUploadRangeFromUriOptions
     {
         public ShareFileUploadRangeFromUriOptions() { }
         public Azure.Storage.Files.Shares.Models.ShareFileRequestConditions Conditions { get { throw null; } set { } }
+        public bool PreserveFileLastWrittenOn { get { throw null; } set { } }
         public Azure.HttpAuthorization SourceAuthentication { get { throw null; } set { } }
     }
     public partial class ShareFileUploadRangeOptions
     {
         public ShareFileUploadRangeOptions() { }
         public Azure.Storage.Files.Shares.Models.ShareFileRequestConditions Conditions { get { throw null; } set { } }
+        public bool PreserveFileLastWrittenOn { get { throw null; } set { } }
```

Comment:
This could expand in the future.  Currently, the service takes a string, with the values of "Preserve" and "Now"